src/utils/getAssetsUrl.test.ts

```diff
@@ -1,10 +1,36 @@
 import { ASSETS_URL } from './constants';
 import { getAssetsUrl } from './getAssetsUrl';
 
+const uploadCareImage = {
+    version: 2,
+    uuid: '43258238-9e71-49fa-a35e-dca5baf16a3b',
+    filename: 'prezly-photographe.png',
+    mime_type: 'image/png',
+    size: 74234,
+    original_width: 1162,
+    original_height: 369,
+    effects: ['/crop/369x369/0,0/', '/preview/'],
+    isImage: true,
+    download_url:
+        'https://cdn.uc.assets.prezly.com/43258238-9e71-49fa-a35e-dca5baf16a3b/-/crop/369x369/0,0/-/preview/-/inline/no/prezly-photographe.png',
+    cdnUrl: 'https://cdn.uc.assets.prezly.com/43258238-9e71-49fa-a35e-dca5baf16a3b/-/crop/369x369/0,0/-/preview/',
+};
+
 describe('getAssetsUrl', () => {
-    it('generates correct asset URL', () => {
-        const uuid = 'test-uuid';
+    it('generates correct asset URL without effects', () => {
+        const image = { ...uploadCareImage, effects: [] };
```

Comment:
This is the only place where we actually need to copy the object, the other two cases can use the object directly, since it's not manipulated in any way.